coursier.bzl

```diff
@@ -30,10 +30,18 @@ package(default_visibility = ["//visibility:{visibility}"])
 
 load("@rules_jvm_external//private/rules:jvm_import.bzl", "jvm_import")
 load("@rules_jvm_external//private/rules:jetifier.bzl", "jetify_aar_import", "jetify_jvm_import")
```

Comment:
Oh wait, should the aar_import in jetifier.bzl be handled the same way?

Reply:
Ah, yes, thank you for pointing that out. It's a little trickier but yes, that will need to be handled too. I'll add some more changes to take care of that.

Reply:
Added support for injecting the aar_import rule to jetifier_aar_import, PTAL